app/src/main/java/org/berlin_vegan/bvapp/activities/LocationsOverviewActivity.java

```diff
@@ -87,7 +87,6 @@
     // menu items
     private MenuItem mFilterItem;
     private MenuItem mMapViewItem;
-    private MenuItem mListViewItem;
```

Comment:
A very reasonable commit, at least with the current state of the source code.
Just mentioning: In the long term I'd love to have a version of the app with both views (list view and map view) as first class citizens: Filter options in the map view etc. ... and then the I'd like to have this ability back to toggle between these views.